### ISSUE TITLE
Python 3 UDF can't return a string

### ISSUE THREAD

**Issue by Oberonhive** [exasol/script-languages#27]
When I trying to run python scalar script that returns single varchar(1000) value, getting error 
`TypeError: in method 'ResultHandler_setString', argument 3 of type 'char *'`

I've used `python3-ds-EXASOL-6.1.0` flavor.

Steps to reproduce:

1. Build and deploy script language
```
git clone https://github.com/exasol/script-languages.git
cd script-languages
sudo ./build --flavor=python3-ds-EXASOL-6.1.0
sudo ./export --flavor=python3-ds-EXASOL-6.1.0
curl -X PUT -T python3-ds-EXASOL-6.1.0.tar.gz http://w:wpwd@localhost:8080/default/lang/py3.tar.gz
```

2. Register script language
```
ALTER SESSION SET SCRIPT_LANGUAGES='PYTHON=builtin_python python3=localzmq+protobuf:///bfsdefault/default/lang/py3?lang=python#buckets/bfsdefault/default/lang/py3/exaudf/exaudfclient';
```
3. Create UDF using registerd language
```
CREATE or replace python3 scalar SCRIPT etl.test() returns varchar(1000) as
def run(ctx):
  return('Test')
/
```
4. Run UDF
```
select etl.test();
```

Output:
```
[22002] VM error: TypeError: Traceback (most recent call last):
File "<EXASCRIPT>", line 136, in emit
TypeError: in method 'ResultHandler_setString', argument 3 of type 'char *'
```
Expected output:
Table with one row 'Test'.

**Comment by stmandl** [comment]
Thx @Oberonhive, I'm going to look into it.

**Comment by stmandl** [comment]
Fixed in commit 5215117a90760a796cfb5208ca5ecadc208bf798

**Comment by stmandl** [comment]
Fixed in 5215117a90760a796cfb5208ca5ecadc208bf798
